core/keyserver/keyserver.go

```diff
@@ -276,7 +277,90 @@ func (s *Server) ListEntryHistory(ctx context.Context, in *pb.ListEntryHistoryRe
 // ListUserRevisions returns a list of revisions covering a period of time.
 func (s *Server) ListUserRevisions(ctx context.Context, in *pb.ListUserRevisionsRequest) (
 	*pb.ListUserRevisionsResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "not implemented")
+	pageStart := in.StartRevision
+	lastVerified := in.GetLastVerifiedTreeSize()
```

Comment:
The `Get` functions are useful if the object before the `.` (`in` in this case) might be `nil`. 
Otherwise, in proto3, all objects have a default value. 

For consistency, consider always using `Get` or trying to save `Get` for cases where it might be needed.